modules/script/script-impl/src/test/resources/graaljs/function-script.js

```diff
@@ -0,0 +1,7 @@
+input => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692791)
```suggestion
(input) => {
```